tests/main/interfaces-microstack-support/test-snapd-sh/bin/inspect

```diff
@@ -0,0 +1,4 @@
+#! /bin/sh
+
+cat /proc/self/cgroup > "$SNAP_COMMON/proc_self_cgroup"
+sleep 120
```

Comment:
since it's a daemon, so why not just `while true; do echo "-- still running"; sleep 1; done` ? it will be stopped when the snaps are purged after the test anyway

Reply:
Makes sense, updated.